notebooks/demo/mg_louvain.ipynb

```diff
@@ -0,0 +1,403 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Multiple GPU Louvain in cuGraph\n",
+    "#### Author : Chuck Hastings\n",
+    "\n",
+    "In this notebook, we will show how to use multiple GPUs in cuGraph to compute the PageRank of each user in Twitter's dataset.\n",
+    "\n",
+    "This notebook was tested using RAPIDS 21.12 and CUDA 11.4. Please be aware that your system may be different, and you may need to modify the code or install packages to run the below examples. If you think you have found a bug or an error, please file an issue in [cuGraph](https://github.com/rapidsai/cugraph/issues)\n",
+    "\n",
+    "\n",
+    "CuGraph's multi-GPU features leverage Dask. RAPIDS has other projects based on Dask such as dask-cudf and dask-cuda. These products will also be used in this example. Check out [RAPIDS.ai](https://rapids.ai/) to learn more about these technologies."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Multi GPU Louvain with cuGraph\n",
+    "### Basic setup"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import needed libraries. We recommend using cugraph_dev env through conda\n",
+    "from dask.distributed import Client, wait\n",
+    "from dask_cuda import LocalCUDACluster\n",
+    "import cugraph.comms as Comms\n",
+    "import cugraph.dask as dask_cugraph\n",
+    "import cugraph\n",
+    "import dask_cudf\n",
+    "import time\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Get the data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The Hollywood dataset is in our S3 bucket and zipped.  \n",
+    "1. We'll need to create a folder for our data in the `/data` folder\n",
+    "1. Download the zipped data into that folder from S3 (it will take some time as it it 6GB)\n",
+    "1. Decompress the zipped data for use (it will take some time as it it 26GB)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Your data file, ../data/hollywood.csv, already exists\n"
+     ]
+    }
+   ],
+   "source": [
+    "import urllib.request\n",
+    "import os\n",
+    "\n",
+    "data_dir = '../data/'\n",
+    "if not os.path.exists(data_dir):\n",
+    "    print('creating data directory')\n",
+    "    os.system('mkdir ../data')\n",
```

Comment:
A more modern way would be to use `pathlib`:
```
import pathlib
pathlib.Path("../data").mkdir()
```
but then you'd be better off updating all the path operations to use `pathlib` for consistency, so if you'd rather stick with the `os` module already imported:
```
os.mkdir("../data")
```

Reply:
We had briefly discussed in slack the idea of making a set of utility functions to support this in a more general way (retrieving test data).  Would you like me to update this, or do you want to defer until we have a better tool?

Reply:
Ah, thanks for the reminder.  I've added [this issue](https://github.com/rapidsai/cugraph/issues/1935) to track it, so feel free to either implement it if you'd like or ignore it in favor of a future utility call.